lib/rex/proto/kerberos/client.rb

```diff
@@ -160,20 +162,41 @@ def send_request_udp(req)
         # @raise [Rex::Proto::Kerberos::Model::Error::KerberosDecodingError] if the response can't be processed
         # @raise [EOFError] if expected data can't be read
         def recv_response_tcp
-          length_raw = connection.get_once(4, timeout)
+          remaining = timeout
```

Comment:
not a blocker: I think this could've used `retry_until_truthy` for improved readability - but maybe the default exponential backoff behavior isn't as useful here 🤔

Reply:
Yeah the exponential backoff would add an unnecessary delay since it wouldn't be reading while it was sleeping before retries.